lib/ensemble_app.dart

```diff
@@ -25,6 +25,7 @@ import 'package:flutter_localizations/flutter_localizations.dart';
 import 'package:workmanager/workmanager.dart';
 
 const String backgroundUploadTask = 'backgroundUploadTask';
+const String ensembleMethodChannelName = 'com.ensembleui.host.platform';
```

Comment:
put this somewhere else. This is the EntryPoint to our app and should have just the minimum possible.

Reply:
should you call it `defaultMethodChannelName`?

Reply:
> should you call it `defaultMethodChannelName`?

Okay, I'll change that, can you tell me where to place this constant?